### DISCUSSION TITLE
Installing on Windows for Python

### DISCUSSION THREAD

**Discussion by prainbow55** [sqlitecloud/discussions#9] [Help] [answered]
I'm learning to code (mainly in Python), not a professional developer.
Trying to install sqlitecloud on Windows 11 according to the github README "Python SDK for SqliteCloud" (https://github.com/sqlitecloud/python?tab=readme-ov-file).
On that page's "Installation" section, right underneath the pip install command, the next line gives a link with "instructions" for building a driver. But on following the link it isn't clear what I have to do to build it. The linked instructions seem designed for app developers using C rather than Python, and all the instructions are for Linux or macOS systems, not Windows.
Everything that follows in the README is clear. That one line (with its linked page) leaves me wondering whether my installation is complete. Do I need to build a driver manually in addition to using pip? If so, what steps?

**Comment by marcobambini** [comment, marked as answer]
@prainbow55 we have updated the Python SDK, which no longer requires compiling C.
Please follow the new instructions because installing now should be much easier.
Let me know if you need any help.
